### ISSUE TITLE
Character's score voting system

### ISSUE THREAD

**Issue by bigfurrypaw** [optc-db/optc-db.github.io#31]
Hello Team, 

Just an idea. =)
There are many characters that I don't know if I need to level up first or not to level them at all. So I am thinking if a voting system like this can be added to your system it would be great. With the huge group of users on this site, I think the judgement will become relatively fair and will benefit many people to utilize their precious time and gems.

5: Stats, Captain or Special. More than two of them have very good use.
4. Stats, Captain or Special. One of them has very good use.
3. Nothing special but good as a backup or lesser replacement.
2. Existing Bat Mission needed. Future Bat mission may be needed. Any use above fodder.
1. Fodder. 

Basically something like the scoring on Gamewith.jp but I think the input should be from all the users.
http://xn--pck6bvfc.gamewith.jp/article/show/2023
Also, I think their scoring is getting too hideous. It can be done simpler and more straight forward. People would just like a general idea of a character's value. 

Thank you for your consideration.
Calvin

**Comment by Hibashira** [comment]
Hi and thanks for your suggestion! This would take quite some changes though. Especially if you take into consideration that whole app is actually running on client side right now. we most likely would have to change that in order to implement a rating system so i cant promise that we will take this on.

**Comment by JNathwani** [comment]
The database was never meant for commenting and voting. If you want to build discussion on a character or team, Gamewith and Reddit are the places to do it.
